xbmc/pvr/channels/PVRRadioRDSInfoTag.cpp

```diff
@@ -117,40 +117,29 @@ bool CPVRRadioRDSInfoTag::operator==(const CPVRRadioRDSInfoTag& right) const
     return true;
 
   std::unique_lock<CCriticalSection> lock(m_critSection);
```

Comment:
nothing special about op==. never saw it before and thought it could be not needed on an `if (tag1 == tag2) {}` .. but yes, it‘s simply a protected read operation.